interface/wx/menuitem.h

```diff
@@ -567,6 +567,25 @@ class wxMenuItem : public wxObject
     */
     virtual void SetAccel(wxAcceleratorEntry *accel);
 
+    /**
+       Add the accel to extra accels list for this item,
+       such accels will not be shown in item's label,
+       but still will work.
+
+       @onlyfor{wxmsw,wxgtk}
+    */
+    void AddExtraAccel(wxAcceleratorEntry *accel);
+
+    /**
+       Return the accels list from this item.
+    */
+    wxVector<wxString> GetExtraAccels() const;
```

Comment:
Not sure if we really need to make this one public, is it actually useful for the application code?

If we do, then it should have `@since 3.1.6` too.